snmp/tests/common.py

```diff
@@ -58,7 +58,7 @@
     {'OID': "1.3.6.1.4.1.2021.10.1.6.1", 'name': "cpuload2"},  # Opaque
 ]
 
-CONSTRAINED_OID = [{"MIB": "RFC1213-MIB", "symbol": "tcpRtoAlgorithm"}]
+CONSTRAINED_OID = [{"MIB": "TCP-MIB", "symbol": "tcpRtoAlgorithm"}]
```

Comment:
> as there is at least one example of breaking behavior when loading conflicting MIBs (like TCP and RFC1213).

What's the breaking behaviour ? I think adding doc/comment might help for later.

Reply:
RFC1213 contains some OIDs from the TCP MIB. I don't have the full background, probably some deprecated path. pysnmp doesn't deal with it properly, because symbols are basically treated as globals, so the test was failing because it was trying to unregister a table row from TCP-MIB that was registered with RFC-1213-MIB. 

Not sure where to document that TBH.

Reply:
I think a docstring for `_get_mib_builder` that explain what you said above might help, with some details on how to reproduce the faulty behaviour in case we need to tackle it later.